Sekoia.io/sekoiaio/triggers/intelligence.py

```diff
@@ -100,9 +107,24 @@ def next_batch(self):
                 level="info",
             )
 
-            # Send objects as result of the trigger
+            # Save objects in file
+            work_dir = self._data_path.joinpath("stix_objects").joinpath(str(uuid.uuid4()))
+            work_dir.mkdir(parents=True, exist_ok=True)
+
+            stix_objects_path = work_dir.joinpath("stix_objects.json")
+            with stix_objects_path.open("w") as fp:
+                fp.write(orjson.dumps(objects).decode("utf-8"))
+
+            # Send event
+            directory = str(work_dir.relative_to(self._data_path))
+            file_path = str(stix_objects_path.relative_to(work_dir))
+            # Attach the file to the event and send objects as result of the trigger
+            event_name = f"Sekoia.io feed - batch of {len(objects)} objects"
             self.send_event(
-                event_name=f"Sekoia.io feed - batch of {len(objects)} objects", event={"stix_objects": objects}
+                event_name=event_name,
+                event=dict(event_path=file_path),
+                directory=directory,
+                remove_directory=True,
             )
```

Comment:
This is a breaking change that will break all the existing playbooks expecting to find a `stix_objects` attribute and not a path